example/snap/snap_example.rb

```diff
@@ -24,6 +24,8 @@
   }
 )
 
+puts result.body
```

Comment:
why printing the `.body`? should print `.data` instead, because in real use case JSON response attribute access is through the `.data[:attribute_name]` right?